### PR TITLE
feat(go-sdk): Adding scaffolded script to generate sample experiment

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by: shubhamchaudhary <shubham.chaudhary@mayadata.io>

- Adding scaffolded script to generate sample experiment

### CODE REVIEW

File: contribute/developer-guide/README.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
+## Steps to Bootstrap a Chaos Experiment
+
+The artefacts associated with a chaos-experiment are summarized below:
```
Comment (chandankumar4) [inline contribute/developer-guide/README.md:3]:
```suggestion
The artifacts associated with a chaos-experiment are summarized below: 
```

File: contribute/developer-guide/README.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
+## Steps to Bootstrap a Chaos Experiment
+
+The artefacts associated with a chaos-experiment are summarized below: 
+
+- Submitted in the litmuschaos/litmus-go repository, under the experiments/*chaos-category* folder i.e, `experiments/generic`
+
+  - Experiment business logic (either as Ansible Playbook, Golang, Python or Bash script, or other). May involve creating new or reusing an existing chaoslib (e.g. pumba, chaostoolkit).
```
Comment (ksatchit) [inline contribute/developer-guide/README.md:7]:
```suggestion
  - Experiment business logic in golang. May involve creating new or reusing an existing chaoslib 
```

File: contribute/developer-guide/README.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+  - variables 
+  - entry & exit criteria checks for the experiment 
+  - helper utils in either [pkg](/pkg/) or new [base chaos libraries](/chaoslib) 
+
+- Update the `experiment.yaml` with the right chaos params in the `spec.definition.env` with their
+  default values
+
+- Update the `chaoslib/litmus/pod-delete/pod-delete.go` chaoslib to achieve the desired effect or reuse the existing chaoslib.
+
+- Create an experiment README (example: [pod delete readme](experiments/generic/pod-delete/README.md)) explaining, briefly,
```
Comment (ksatchit) [inline contribute/developer-guide/README.md:144]: remove link

File: contribute/developer-guide/templates/experiment_custom_resource.tmpl [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    {{ description }}
+kind: ChaosExperiment
+metadata:
+  name: {{ name }}
+  version: {{ version }} 
+spec:
+  definition:
+    scope: {{ scope }}
+    permissions: {{ permissions }}
+    image: "litmuschaos:litmuschaos-go:latest"
```
Comment (ksatchit) [inline contribute/developer-guide/templates/experiment_custom_resource.tmpl:13]:
```suggestion
    image: "litmuschaos/go-runner:latest"
```

**Pull request merged**